src/elements/load.ts

```diff
@@ -46,20 +59,75 @@ const loadElements = (
           script = injectScript(url);
         }
 
-        script.addEventListener('load', () => {
+        // listen for window error events for same script
+        // eslint-disable-next-line unicorn/prefer-add-event-listener
+        window.onerror = (message, source, lineno, colno, error) => {
+          telemetryLogger.logger.error(
+            'Elements script onError event from window',
+            {
+              logType: 'elementsNotFoundOnWindow',
+              logOrigin: 'loadElements',
+              errorObject: error,
+              error: {
+                message,
+                source,
+                lineno,
+                colno,
+                error,
+              },
+            }
+          );
+        };
```

Comment:
Would this cause issues to other parties that rely on `window.onerror`?

Reply:
it only reports errors coming from same origin